packages/@o3r/eslint-config-otter/schematics/ng-add/linter/templates/tsconfig.eslint.json

```diff
@@ -1,9 +1,8 @@
 {
-  "extends": "./tsconfig.app",
+  "extends": "./tsconfig.json",
   "include": [
     ".eslintrc.js",
     "jest.config.js",
-    "src/**/*.ts",
-    "e2e-playwright/**/*.ts"
+    "**/src/**/*.ts"
```

Comment:
Not needed as well

Reply:
It is actually needed
```shell
  0:0  error  Parsing error: ESLint was configured to run on `<tsconfigRootDir>/apps/webapp/src/main.ts` using `parserOptions.project`: test-eslint-basic-2/tsconfig.eslint.json
However, that TSConfig does not include this file. Either:
- Change ESLint's list of included files to not include this file
- Change that TSConfig to include this file
- Create a new TSConfig that includes this file and include it in your parserOptions.project
See the typescript-eslint docs for more info: https://typescript-eslint.io/linting/troubleshooting#i-get-errors-telling-me-eslint-was-configured-to-run--however-that-tsconfig-does-not--none-of-those-tsconfigs-include-this-file
```

Reply:
This error is different, it just mean that you should register both `tsconfig.json` and `tsconfig.eslint.json` in the ts-eslint projects field

Reply:
`tsconfig.json` doesn't have any path mapping. Let's discuss this point tomorrow morning.